test_resources/test_user_agent_parser_os.yaml

```diff
@@ -2029,3 +2029,10 @@ test_cases:
     patch: '2'
     patch_minor:
 
+
+  - user_agent_string: 'SoundCloud-Android/14.08.21-17 (Android 4.4.2; samsung SAMSUNG-SGH-I337)'
```

Comment:
This was changed [here](https://github.com/soundcloud/android-listeners/pull/7706) recently. We are sending the following user agent:
`SoundCloud/2017.04.18-release (Android 4.1.1; Samsung GT-I9082)`

Notice how the first part (the tag) is not `SoundCloud-Android` anymore, but instead, just `SoundCloud`.
I've added one test to verify it on #5.